.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
+contact_links:
+  - name: OpenTabletDriver Discord Guild
+    url: https://discord.gg/9bcMaPkVAR
```

Ideally would point to https://opentabletdriver.net/Discord so that it always uses our primary link, even though this just redirects to that.
```suggestion
    url: https://opentabletdriver.net/Discord
```